features/topology-map/plugins/org.opennms.features.topology.plugins.topo.application/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -17,6 +17,9 @@ xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.o
 	<!-- Make it hop aware -->
 	<bean id="applicationTopologyHopProvider" class="org.opennms.features.topology.api.support.VertexHopGraphProvider">
 		<argument ref="applicationTopologyProvider" />
+
+		<property name="name" value="Application Topology Provider"/>
+		<property name="description" value="This Topology Provider displays the defined application topologies."/>
```

Comment:
Maybe `This Topology Provider displays all defined Applications and its calculated state`